config/metadata/configMeta.yaml

```diff
@@ -542,7 +539,8 @@ groups:
         reload: false
         validations:
           - type: requiredInGroup
-        envvar: REFINERY_METRICS_API_KEY
+        envvar: REFINERY_LEGACY_METRICS_API_KEY, HONEYCOMB_API_KEY
```

Comment:
Do we need `LEGACY` in the name?  Also, that is a breaking change we want to make sure we document because it can't be handled by the converter tool, users would need to update their pipelines